packages/standard-generator/src/generator.config.js

```diff
@@ -74,6 +74,7 @@ module.exports = {
        */
       modulePath: '@jpmorganchase/mosaic-source-git-repo',
       namespace: 'mosaic', // each site has it's own namespace, think of this as your content's uid
+      disabled: !process.env.MOSAIC_DOCS_CLONE_CREDENTIALS, // use a boolean to enable/disable the source
```

Comment:
The simplest possible site, is starting up with just local docs.
You cannot start with that, as Mosaic will fail to start if `MOSAIC_DOCS_CLONE_CREDENTIALS` is undefined

Use this to toggle the boolean, so that we can progressively enhance the local setup to load local docs and then both local and remote docs.

As the initial startup is the welcome mat for new users, we should not start their experience with an error, which they may not yet understand.